src/Components/Artwork/Details.tsx

```diff
@@ -139,7 +139,7 @@ const HighDemandInfo = () => {
   return (
     <Flex flexDirection="row" alignItems="center">
       <HighDemandIcon width={16} height={16} />
-      <Text variant="xs" color="blue100" ml={0.5}>
+      <Text variant="xs" color="blue100" ml={0.3}>
```

Comment:
That's not a valid value

Reply:
`0.5, 1, 2, 4, 6, 12` — that's it.

Reply:
Thanks! I guess, in this case, it's ok to not use the fixed values from Palette and changed it to "3px" in https://github.com/artsy/force/pull/11358.